test/parallel/test-cli-eval.js

```diff
@@ -19,13 +19,15 @@ const filename = __filename.replace(/\\/g, '/');
 // assert that nothing is written to stdout
 child.exec(nodejs + ' --eval 42',
            function(err, stdout, stderr) {
+             assert.strictEqual(err, null);
```

Comment:
These will work and are totally fine, but it's probably more idiomatic to do this:

```js
assert.ifError(err);
```